lib/foreman_maintain/reporter/cli_reporter.rb

```diff
@@ -117,7 +117,11 @@ def ask(message, options = {})
         # add space at the end as otherwise highline would add new line there :/
         message = "#{message} " unless message =~ /\s\Z/
         answer = @hl.ask(message) { |q| q.echo = false if options[:password] }
-        answer.to_s.chomp.downcase if answer
+        if answer
+          answer.to_s.chomp!
+          answer.downcase! unless options[:username] || options[:password]
```

Comment:
How does the `:username` option even get here? Notice, that we validate what options we are accepting on line 113.

I think we should remove the downcase here, and use it in `filter_decision` instead (+ check other places we use `ask` to make sure we have not missed something.

Reply:
Yes, right! I have missed it.
Only for password, passed options `password` to `ask` method.